udata_transport/settings.py

```diff
@@ -0,0 +1,6 @@
+'''
+Default settings for udata-transport
+'''
+
+
+TRANSPORT_DATASETS_URL = 'https://transport.data.gouv.fr/api/datasets'
```

Comment:
Did we check for any info on pagination?

Reply:
Waiting for answer